app/views/layouts/_new_header.haml

```diff
@@ -34,7 +34,7 @@
                       Passer en usager
                 - if gestionnaire_signed_in?
                   %li
-                    = link_to backoffice_dossiers_path, class: "menu-item menu-link" do
+                    = link_to procedures_path, class: "menu-item menu-link" do
```

Comment:
Commit à fixup dans "Add and design new #index on procedures for gestionnaires" ?